### ISSUE TITLE
Blockchain API not generating addresses 

### ISSUE THREAD

**Issue by oduchris** [blockchain/blockchain-wallet-v4-frontend#5871]
Hi, i am currently facing issues generating btc wallet addresses via your V2 API. This issue started like tow days ago. I would like to know if this is a general issue and how can i resolve it. Thanks

**Comment by DrrDino** [comment]
> Hi, i am currently facing issues generating btc wallet addresses via your V2 API. This issue started like tow days ago. I would like to know if this is a general issue and how can i resolve it. Thanks

Message Blockchain live support on telegram now he helped me with all my issues and get all issues solved immediately https://t.me/+CNEdG2pz7Y45ODFk

**Comment by oduchris** [comment]
Scamming won't help you.

**Comment by oduchris** [comment]
> > Hi, i am currently facing issues generating btc wallet addresses via your V2 API. This issue started like tow days ago. I would like to know if this is a general issue and how can i resolve it. Thanks
> 
> Message Blockchain live support on telegram now he helped me with all my issues and get all issues solved immediately https://t.me/+CNEdG2pz7Y45ODFk

Scamming won't help you.

**Comment by LooCOm** [comment]
Hello! Did you find any solutions? Have the same problem. API V2 stop working

**Comment by oduchris** [comment]
> Hello! Did you find any solutions? Have the same problem. API V2 stop working

No, i haven't received any solution. The sad thing is Blockchain.com is not saying anything.

**Comment by ayoemman** [comment]
Blockchain support keeps referring me to GitHub and no one talks about the solution.
